diff --git a/Telegram/SourceFiles/app.cpp b/Telegram/SourceFiles/app.cpp
index 45efcad4..30d38843 100644
--- a/Telegram/SourceFiles/app.cpp
+++ b/Telegram/SourceFiles/app.cpp
@@ -7,6 +7,8 @@ https://github.com/telegramdesktop/tdesktop/blob/master/LEGAL
 */
 #include "app.h"
 
+#include "../../config.h"
+
 #ifdef OS_MAC_OLD
 #include <libexif/exif-data.h>
 #endif // OS_MAC_OLD
@@ -2092,6 +2094,7 @@ namespace {
 	void initMedia() {
 		if (!::monofont) {
 			QString family;
+			tryFontFamily(family, TELEGRAM_FT_MONOSPACE_OVERRIDE);
 			tryFontFamily(family, qsl("Consolas"));
 			tryFontFamily(family, qsl("Liberation Mono"));
 			tryFontFamily(family, qsl("Menlo"));
diff --git a/Telegram/SourceFiles/ui/twidget.cpp b/Telegram/SourceFiles/ui/twidget.cpp
index 1441ea47..63c51e92 100644
--- a/Telegram/SourceFiles/ui/twidget.cpp
+++ b/Telegram/SourceFiles/ui/twidget.cpp
@@ -10,6 +10,8 @@ https://github.com/telegramdesktop/tdesktop/blob/master/LEGAL
 #include "application.h"
 #include "mainwindow.h"
 
+#include "../../../config.h"
+
 namespace Fonts {
 namespace {
 
@@ -59,8 +61,8 @@ bool LoadCustomFont(const QString &filePath, const QString &familyName, int flag
 }
 
 bool Started = false;
-QString OpenSansOverride;
-QString OpenSansSemiboldOverride;
+QString OpenSansOverride         = TELEGRAM_FT_OVERRIDE;
+QString OpenSansSemiboldOverride = TELEGRAM_FT_SEMIBOLD_OVERRIDE;
 
 } // namespace
 
